spec/service_models/miq_ae_service_service_template_transformation_plan_task_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe MiqAeMethodService::MiqAeServiceServiceTemplateTransformationPlanTask do
+  let(:user) { FactoryGirl.create(:user_with_group) }
+  let(:task) { FactoryGirl.create(:service_template_transformation_plan_task) }
+
+  before(:each) do
+    Spec::Support::MiqAutomateHelper.create_service_model_method('SPEC_DOMAIN', 'EVM', 'AUTOMATE', 'test1', 'test')
+    @ae_method = ::MiqAeMethod.first
+  end
+
+  def invoke_ae
+    MiqAeEngine.instantiate("/EVM/AUTOMATE/test1?ServiceTemplateTransformationPlanTask::service_template_transformation_plan_task=#{task.id}", user)
+  end
+
+  describe "#conversion_host=" do
+    before(:each) do
+      @conversion_host = FactoryGirl.create(:conversion_host)
+    end
+
+    it "removes the conversion host if arg is nil" do
+      task.conversion_host = @conversion_host
+      method = "$evm.root['#{@ae_result_key}'] = $evm.root['service_template_transformation_plan_task'].conversion_host = nil"
```

Comment:
@fdupont-redhat I would prefer we avoid the overhead of instantiation the engine to test this, you should be able to create the active_record and service_model objects and test directly.

There are several tests that can be used as an example.  Here's one: https://github.com/ManageIQ/manageiq-automation_engine/blob/master/spec/service_models/miq_ae_service_orchestration_stack_spec.rb

Reply:
@gmcculloug Done. My previous tests were inspired by https://github.com/ManageIQ/manageiq-automation_engine/blob/master/spec/service_models/miq_ae_service_vm_spec.rb#L75.